custom_components/adaptive_lighting/switch.py

```diff
@@ -2009,12 +1994,6 @@ def modify_service_data(service_data, entity_ids):
                 context.id,
             )
             service_data = {ATTR_ENTITY_ID: skipped, **service_data_copy[CONF_PARAMS]}
-            if (
-                ATTR_COLOR_TEMP in service_data
-                and ATTR_COLOR_TEMP_KELVIN in service_data
-            ):
-                # ATTR_COLOR_TEMP and ATTR_COLOR_TEMP_KELVIN are mutually exclusive
-                del service_data[ATTR_COLOR_TEMP]
```

Comment:
Why can this code be deleted?

Reply:
`ATTR_COLOR_TEMP` is now deprecated and services are now using only `ATTR_COLOR_TEMP_KELVIN`. Therefore it is not required anymore.